pandas/io/json/_json.py

```diff
@@ -102,6 +104,8 @@ def to_json(
             fh.write(s)
         finally:
             fh.close()
+        for handle in handles:
```

Comment:
was there a reason not to close them?

Reply:
could have been an oversight, did you have to change any tests?

Reply:
I didn't change any tests